doozer/doozerlib/backend/konflux_image_builder.py

```diff
@@ -281,6 +281,8 @@ async def _get_for_arch(arch):
             return source_rpms
 
         results = await asyncio.gather(*(_get_for_arch(arch) for arch in arches))
+        if None in results:
```

Comment:
I had to check this, because it feels wrong, but it is exactly right:

```python
>>> a = [1, 'a']
>>> None in a
False
>>> a = [1, 'a', None]
>>> None in a
True
>>> a = [1, 'a', []]
>>> None in a
False
>>> a = [1, 'a', '']
>>> None in a
False
```